eftcamb/06p1_abstract_EFTCAMB_full_map.f90

```diff
@@ -68,8 +68,9 @@ function EFTCAMBFullModelComputeDtauda( self, a, eft_par_cache, eft_cache )
 
         real(dl) :: EFTCAMBFullModelComputeDtauda                     !< the output dtauda
 
-        write(*,*) 'IW'
-        stop
+        call self%compute_adotoa( a, eft_par_cache, eft_cache )
+
+        EFTCAMBFullModelComputeDtauda = 1.0_dl/sqrt( a**2*eft_cache%adotoa**2 )
```

Comment:
Something like this:

    dtauda = 1._dl/sqrt( ( grho_matter_a2/3._dl +2._dl/3._dl*Omega_Lambda_c(a,0)*a2 + a2*EFT_H0**2*CP%omegav*a2*(1._dl +EFT_OL_Lambda(a,0) ) )/( 1._dl +EFT_OL_Omega(a,0) +a*EFT_OL_Omega(a,1) ) )